### PR TITLE
Ksy

### PR DESCRIPTION
**Pull request by KSY1526**
1. 알림이 정상적으로 왔으면 확인 댓글 부탁
2. python train.py -c config.json 코드를 쳤을 때 돌아가는지 확인
3. 댓글 모두 쓰면 main에 merge 할 것.
4. merge후 본인 작업물 master에서 pull로 땡긴 뒤 push로 업로드 해주면 좋겠음.
5. 피어세션때 보자구

### CODE REVIEW

File: model/loss.py [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import torch.nn.functional as F
-
+import torch.nn as nn
 
 def nll_loss(output, target):
     return F.nll_loss(output, target)
+
+def movie_loss(output, target):
+    _loss = nn.BCELoss()
```
Comment (sungsubae) [inline model/loss.py:8]: 이거 cuda로 돌려도 잘 돌아가? 과제코드에서는 .to(device)가 있길래 궁금하네.

**Review by sungsubae — COMMENTED** [review]
확인했어~

**Comment by KChanho** [comment]
✔️ 확인~

File: model/loss.py (hunk 1 above)
Comment (KSY1526) [reply model/loss.py:8]: 원래 로스함수에 난 device를 안쓰는데 과제에는 있더라구.
없어도 돌아가긴 하는데 정확힌 모르겠네
꼼꼼히 봐줘서 고마워~

**Comment by KSY1526** [comment]
> ✔️ 확인~



>

**Comment by KSY1526** [comment]
> ✔️ 확인~

굿

**Comment by JJI-Hoon** [comment]
확인~~~

**Comment by hxcva1** [comment]
확인!


2022년 10월 21일 (금) 오후 2:14, Seong Yeon Kim ***@***.***>님이 작성:
